modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -25,53 +25,69 @@ apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
   name: a-bridge-network
-  annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
 spec:
   config: '{
     "cniVersion": "0.3.1",
-    "name": "cnv-bridge-conf", <2>
+    "name": "cnv-bridge-conf", <1>
     "plugins": [
       {
-        "type": "cnv-bridge", <3>
-        "bridge": "br0" <4>
+        "type": "cnv-bridge", <2>
+        "bridge": "br0" <3>
       },
       {
-        "type": "cnv-tuning" <5>
+        "type": "cnv-tuning" <4>
       }
     ]
   }'
 ----
-<1> If you add this annotation to your NetworkAttachmentDefinition, your virtual machine instances
-will only run on nodes that have the `br0` bridge connected.
-<2> Required. A name for the configuration.
-<3> The actual name of the Container Network Interface (CNI) plug-in that provides
+<1> Required. A name for the configuration.
+<2> The actual name of the Container Network Interface (CNI) plug-in that provides
 the network for this NetworkAttachmentDefinition. Do not change this field unless
 you want to use a different CNI.
-<4> You must substitute the actual name of the bridge, if it is not `br0`.
-<5> Required. This allows the MAC pool manager to assign a unique MAC address to the connection.
+<3> You must substitute the actual name of the bridge, if it is not `br0`.
+<4> Required. This allows the MAC pool manager to assign a unique MAC address to the connection.
++
+[NOTE]
+====
+The virtual machine instance will be attached to the bridge `br0` through an
+access port with the requested VLAN.
+====
 +
 ----
 $ oc create -f <resource_spec.yaml>
 ----
 
-. Edit the configuration file of a virtual machine or virtual machine instance that you want to connect to the
-bridge network:
+. Edit the virtual machine instance configuration file to include the details of the interface and network.
+
+.. Specify the network and MAC address, if required by the server. If the MAC address is not specified, a value is assigned
+automatically. However, note that at this time, MAC addresses assigned automatically are not persistent.
 +
-[source,yaml]
 ----
-apiVersion: v1
-kind: VirtualMachine
-metadata:
-  name: example-vm
-  annotations:
-    k8s.v1.cni.cncf.io/networks: a-bridge-network <1>
-spec:
-...
+interfaces:
+- masquerade: {}
```

Comment:
Worth a note that this VM will be connected to both the default pod network and to the bridge.